pkg/model/message.go

```diff
@@ -14,6 +14,29 @@ type Messages struct {
 	Original bool
 }
 
+type MessagesSlice []Messages
+
+// GetCopy() returns a deep copy of MessagesSlice.
+func (m MessagesSlice) GetCopy() MessagesSlice {
+	if m == nil {
+		return nil
+	}
+
+	msgs := make(MessagesSlice, len(m))
+
+	for i := range m {
+		msgs[i] = Messages{
+			Language: m[i].Language,
+			Messages: make([]Message, len(m[i].Messages)),
+			Original: m[i].Original,
+		}
+
+		copy(msgs[i].Messages, m[i].Messages)
+	}
+
+	return msgs
+}
```

Comment:
As we don't have any references inside the Messages struct, we can make a shallow copy.

And shallow copy can be made using standard lib (go 1.21) slices.Clone

So this method becomes redundant

Reply:
We have a reference to the messages slice, it will get modified if shallow copy is used.